### PR TITLE
Update the README of Data Connect to help developers onboarding

### PR DESCRIPTION
**Pull request by charlotteliang**

### CODE REVIEW

**Comment by charlotteliang** [comment]
First version, will add SDK part later.

**Comment by charlotteliang** [comment]
Not sure the check is valid for the PR.

**Comment by paulb777** [comment]
You can ignore the CI failure. It will be fixed by the next rebase or merge from `main`

File: FirebaseDataConnect/README.md [hunk 1]
```diff
@@ -1,19 +1,126 @@
...
+* Select the Firebase icon on the left and Click on the "Deploy all" button to deplouy all the schema and operations to backend.
+
+### 7. Set up generated SDK location
+In the connector.yaml file, add the following code to enable swift code to be generated.
+
+```
+  swiftSdk:
+     outputDir: "../swift-generated/"
+     package: "User"
+     coreSdkPackageLocation: "file:///Users/[YOURUSERID]/github/firebase-ios-sdk"
```
Comment (cherylEnkidu) [inline FirebaseDataConnect/README.md:76]: The team is schedule a new version of cli release today, so the developer don't need to include this line `coreSdkPackageLocation:` in yaml file.

**Review by cherylEnkidu — CHANGES_REQUESTED** [review]
Thank you for the instruction!

File: FirebaseDataConnect/README.md [hunk 2]
```diff
@@ -1,19 +1,126 @@
...
+
+### 2. Set up [Firebase CLI](https://firebase.devsite.corp.google.com/docs/cli)
+
+* If you already have CLI, make sure you always update to the latest version
+```
+npm install -g firebase-tools
+```
+
+### 3. Set up VSCode
+You will need VS Code and its Firebase extension (VS Code extension) to automatically generate Swift code for your queries.
```
[the remark this answers is not in the document]
Comment (charlotteliang) [reply FirebaseDataConnect/README.md:24]: The VS Code extension includes the emulator toolkit.

File: FirebaseDataConnect/README.md [hunk 3]
```diff
@@ -1,19 +1,126 @@
...
+* Install VS Code
+* Download the [extension](https://firebasestorage.googleapis.com/v0/b/firemat-preview-drop/o/vsix%2Ffirebase-vscode-latest.vsix?alt=media) and drag it into the "Extensions" Left Nav bar for installation.
+* Create a fdc folder
+```
+mkdir fdc
+```
+* Open VS Code from FDC folder
```
[the remark this answers is not in the document]
Comment (charlotteliang) [reply FirebaseDataConnect/README.md:31]: Thats the folder we suggested to make, maybe I will adjust the above step to: create a folder you'd like to have firebase configuration

File: FirebaseDataConnect/README.md (hunk 1 above)
Comment (charlotteliang) [reply FirebaseDataConnect/README.md:76]: Currently the default option doesn't do anything setting the path, we have a meeting this afternoon to discuss so I think we still need this.

File: FirebaseDataConnect/README.md [hunk 4]
```diff
@@ -1,19 +1,126 @@
...
+
+* If you already have CLI, make sure you always update to the latest version
+```
+npm install -g firebase-tools
+```
+
+### 3. Set up VSCode
+You will need VS Code and its Firebase extension (VS Code extension) to automatically generate Swift code for your queries.
+* Install VS Code
+* Download the [extension](https://firebasestorage.googleapis.com/v0/b/firemat-preview-drop/o/vsix%2Ffirebase-vscode-latest.vsix?alt=media) and drag it into the "Extensions" Left Nav bar for installation.
```
[the remark this answers is not in the document]
Comment (charlotteliang) [reply FirebaseDataConnect/README.md]: Done.

File: FirebaseDataConnect/README.md [hunk 5]
```diff
@@ -1,19 +1,126 @@
...
+
+Firebase Data Connect is in Private Preview at no cost for a limited time. Sign up the program at https://firebase.google.com/products/data-connect.
+
+Once you are selected as an allowlist member, you should be able to create a Cloud SQL instance through Firebase Data Connect console.
+
+## Getting Started
+Here's a quick rundown of steps to get you started. Learn more about details at the official [Getting Started documentation](https://firebase.google.com/docs/data-connect/quickstart).
+
+### 1. Create a new Data Connect service and Cloud SQL instance.
+* Go to Firebase Console and select Firebase Data Connect from LeftNav bar to create a new Data Connect service and a Cloud SQL instance. You have to be in Blaze plan and you can view the details of pricing at https://firebase.google.com/pricing.
```
[the remark this answers is not in the document]
Comment (charlotteliang) [reply FirebaseDataConnect/README.md]: Done.

**Review by cherylEnkidu — APPROVED** [review]

**Pull request merged**